helper/user_object.js

```diff
@@ -0,0 +1,29 @@
+const UserDocumentMetadata = {
+  is_public: false,
+  shared_with: {
+    uuid: false,
+  },
+};
+
+const UserFriendMetadata = {
+  shared_duid_to_me: {
+    duid: false,
+  },
+};
+
+const User = {
+  uuid: '',
+  first_name: '',
+  last_name: '',
+  email: '',
+  password: '',
+  organization: '',
+  user_documents: {
+    duid: UserDocumentMetadata,
+  },
+  shared_to_me: {
+    uuid: UserFriendMetadata,
+  },
+};
+
+export default User;
```

Comment:
consider using factory pattern.
https://medium.com/@SntsDev/the-factory-pattern-in-js-es6-78f0afad17e9